ert3/evaluator/_evaluator.py

```diff
@@ -1,25 +1,113 @@
 import asyncio
-import os
 import pathlib
 import shutil
 from collections import defaultdict
 from pathlib import Path
-from typing import Dict, Any, Tuple, List
+from typing import Coroutine, Dict, Any, Tuple, List
 
 
 import cloudpickle
-from pydantic import FilePath
 
 import ert3
 from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
 from ert_shared.ensemble_evaluator.evaluator import EnsembleEvaluator
 from ert_shared.ensemble_evaluator.prefect_ensemble import PrefectEnsemble
-
-from ert3.config import EnsembleConfig, StagesConfig, Step
-from ert3.data import EnsembleRecord, MultiEnsembleRecord, RecordTransmitter, Record
+from ert3.config import (
+    EnsembleConfig,
+    StagesConfig,
+    TransportableCommand,
+    Unix,
+    Function,
+    Step,
+)
+from ert3.data import (
+    EnsembleRecord,
+    MultiEnsembleRecord,
+    RealisationsToRecordToTransmitter,
+    RecordTransmitter,
+    Record,
+    SharedDiskRecordTransmitter,
+    record_data,
+)
 
 _EVTYPE_SNAPSHOT_STOPPED = "Stopped"
 _EVTYPE_SNAPSHOT_FAILED = "Failed"
+_MY_STORAGE_DIRECTORY = ".my_storage"
+
+CoroutineTransmitters = List[Coroutine[Any, Any, None]]
+
+
+# scafflting function. this will be removed later
+# with ert3.evaluator._evaluator._ee_config
+def _add_storage(path: Path) -> Path:
+    if path.name != _MY_STORAGE_DIRECTORY:
+        return path / _MY_STORAGE_DIRECTORY
+    return path
+
+
+def _create_input_transmitter(
+    name: str,
+    path: Path,
+    data: record_data,
+    futures: CoroutineTransmitters,
+) -> Dict[str, RecordTransmitter]:
+    transmitter = SharedDiskRecordTransmitter(name, _add_storage(path))
+    futures.append(transmitter.transmit_data(data))
+    return {name: transmitter}
```

Comment:
I would recommend removing this function, it does not do much, it is only used in one place and also it has a side-effect and we should avoid them.